blocktx/store/sql/get_block_test.go

```diff
@@ -17,7 +17,7 @@ type GetBlockTestSuite struct {
 	DatabaseTestSuite
 }
 
-func (s *GetBlockByHeightTestSuite) TestGetBlock() {
+func (s *GetBlockTestSuite) Test() {
```

Comment:
Here it is called `Test()` but everywhere else it is called `Run()`

Reply:
renamed all the methods to `Test`.